FILE-STORAGE.md

```diff
@@ -51,3 +73,8 @@ Media files are not inherently public. For media assets that require access cont
 eg. See `def profile_photo`.
 
 With GCS uploads, it is possible to link directly to the GCS location in the template with: `<a src="{{ <model>.<fieldname>.url}}">`. Django automatically generates a signed URL for the GCS location. Only do this if the media asset should be publicly accessible.
+- Create an IAM user with programmatic access, and attach the `AmazonS3FullAccess` policy.
```

Comment:
not sure - why is this talking about AmazonS3 and various AWS keys?

Reply:
Fixed.